packages/react-reconciler/src/ReactFiberThenable.js

```diff
@@ -212,19 +212,19 @@ export function trackUsedThenable<T>(
             }
           },
         );
+      }
```

Comment:
@acdlite I believe that the end curly was not in the right position because if we hit this path with an uninitialized Flight object, we should be able to synchronously return the value without suspending once.

https://github.com/facebook/react/blob/main/packages/react-reconciler/src/ReactFiberThenable.js#L164-L169